src/ansible/deploy.yaml

```diff
@@ -63,11 +62,16 @@
       shell: |
         nohup /opt/rocm/rdc/bin/rdcd -u </dev/null >/dev/null 2>&1 &
       when: amd_gpu_exist
-      become: yes
+      become: yes      
```

Comment:
Are we leaving unnecessary spaces here?

Reply:
let me address this specific issue. At some point I will need to go through the code base and look for these. Like with lint or something. A bit out of scope for this PR at the moment.